apps/molgenis-components/lib/main.js

```diff
@@ -47,12 +53,28 @@ import ObjectDisplay from "../src/components/tables/cellTypes/ObjectDisplay.vue"
 import StringDisplay from "../src/components/tables/cellTypes/StringDisplay.vue";
 import TextDisplay from "../src/components/tables/cellTypes/TextDisplay.vue";
 
-import SignInForm from "../src/components/organisms/SignInForm.vue"
-
+import MolgenisSignIn from "../src/components/layout/MolgenisSignIn.vue";
+import ShowMore from "../src/components/layout/ShowMore.vue";
```

Comment:
Placing basic components into the same very large folder as the compositions makes it hard to locate and navigate components. In addition leading with the non descriptive ( and inconsistent ) 'Molgenis' prefix makes it harder to scan the components tree

Reply:
I do agree but this currently we try to keep close to styleguide. Please leave unnecessary refactorings to later.